contribs/drt/src/main/java/org/matsim/contrib/drt/analysis/DrtEventSequenceCollector.java

```diff
@@ -60,31 +65,35 @@
 public class DrtEventSequenceCollector
 		implements PassengerRequestRejectedEventHandler, PassengerRequestScheduledEventHandler,
 		DrtRequestSubmittedEventHandler, PassengerPickedUpEventHandler, PassengerDroppedOffEventHandler,
-		PersonMoneyEventHandler {
+		PersonMoneyEventHandler, PersonDepartureEventHandler {
 
-	private static final Logger log = Logger.getLogger(DrtEventSequenceCollector.class);
-
-	public static class PerformedRequestEventSequence {
+	public static class EventSequence {
 		private final DrtRequestSubmittedEvent submitted;
-		private final PassengerRequestScheduledEvent scheduled;
-		//pickedUp and droppedOff may be null if QSim ends before the request is actually handled
+		
+		@Nullable
+		private PassengerRequestScheduledEvent scheduled;
+		@Nullable
+		private PassengerRequestRejectedEvent rejected;
+		
+		@Nullable
+		private PersonDepartureEvent departed;
```

Comment:
Rename to `departure`? Otherwise, it sounds like we need to send a caravan for that person :-)